packages/desktop-client/src/components/tooltips.tsx

```diff
@@ -233,7 +280,13 @@ export class Tooltip extends Component {
   }
 
   getStyleForPosition(position, boxRect, anchorRect, containerRect, offset) {
-    const style = {
+    const style: {
+      top?: string;
+      bottom?: string;
+      left?: string;
+      right?: string;
+      width?: string;
+    } = {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏don't think we need these as they can be automatically inferred from the values we define below.